python/fledge/services/core/support.py

```diff
@@ -67,10 +69,26 @@ def __init__(self, support_dir):
             try:
                 self.add_syslog_fledge(pyz, file_spec)
                 self.add_syslog_storage(pyz, file_spec)
+                cf_mgr = ConfigurationManager(self._storage)
+                try:
+                    south_cat = await cf_mgr.get_category_child("South")
+                    south_categories = [sc["key"] for sc in south_cat]
+                    for service in south_categories:
+                        self.add_syslog_service(pyz, file_spec, service)
+                except:
+                    pass
+                try:
+                    north_cat = await cf_mgr.get_category_child("North")
+                    north_categories = [nc["key"] for nc in north_cat]
```

Comment:
`"key": "OMF_TYPES"` also have a children of North, I see related syslog file has been created with empty content. We may skip this key.